### DISCUSSION TITLE
What is the difference between config.js and renovate.json?

### DISCUSSION THREAD

**Discussion by belek** [renovatebot/renovate#28094] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

BitBucket

### Please tell us more about your question or problem

Hi! I am a little confused about purpose of config.js and renovate.json files. Could someone please explain?

I understand that config.js is a global config, where I configure renovate behavior. But I don't understand what should be in renovate.json. So far I found out that I can set enabledManagers in desired renovate.json for specific repository where I would like to renovate work with other package managers. 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Here is an in-depth guide to Renovate config: https://github.com/renovatebot/renovate/blob/docs/config-overview/docs/usage/config-overview.md

If you are both the admin and only user of your Renovate, and have one repository, then it doesn't matter too much where you put your Repository config - you could put it all in global config. But as soon as you grow repositories or have other users and need different config for different repositories, you will want to put as much of config as possible in Repository config.

**Reply by ivan-zaitsev** [reply]
Is it possible to create renovate.json per project in separate repository or in the same repo where Renovate config is stored? I don't want project repo to be coupled to pipeline specific stuff and instead decouple it in separate repo.

**Reply by rarkins** [reply]
It's always per repo it clones and never specific to the pipeline it runs in
